packages/react-charting/src/components/HorizontalBarChart/HorizontalBarChart.types.ts

```diff
@@ -88,6 +88,18 @@ export interface IHorizontalBarChartProps {
    * @default false
    */
   hideLabels?: boolean;
+
+  /**
+   * Prop to set the x axis title
+   * @default undefined
+   */
+  xAxisTitle?: string;
+
+  /**
+   * Prop to set the y axis title
+   * @default undefined
+   */
+  yAxisTitle?: string;
```

Comment:
Horizontal chart type does not have a y axis.

Reply:
removed and kept only in cartesian base